packages/desktop-client/src/components/select/DateSelect.js

```diff
@@ -45,28 +45,33 @@ let pickerStyles = {
       float: 'none',
       width: 'auto',
     },
+    // month/year
     '& .pika-label': {
-      backgroundColor: colors.n1,
+      backgroundColor: theme.sidebarBackground,
     },
+    // Back/forward buttons
     '& .pika-prev': {
       backgroundImage: `url(${DateSelectLeft})`,
     },
     '& .pika-next': {
       backgroundImage: `url(${DateSelectRight})`,
     },
+    // Day of week
     '& .pika-table th': {
-      color: colors.n11,
+      color: theme.sidebarItemText,
       '& abbr': { textDecoration: 'none' },
     },
+    // Numbered days
     '& .pika-button': {
-      backgroundColor: colors.n2,
-      color: colors.n11,
+      backgroundColor: theme.sidebarItemBackgroundHover,
+      color: theme.sidebarItemText,
     },
     '& .is-today .pika-button': {
       textDecoration: 'underline',
     },
     '& .is-selected .pika-button': {
-      backgroundColor: colors.n5,
+      backgroundColor: theme.buttonNormalSelectedBackground,
+      color: theme.buttonNormalSelectedText,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏wrong color on lightmode